www/_layouts/base.html

```diff
@@ -3,12 +3,20 @@
 ---
 
 <!DOCTYPE html>
-<html lang="{% if page.language %}{{ page.language }}{% else %}{{ site.language }}{% endif %}">
+{% if page.language %}
+    {% assign language = page.language %}
+{% else %}
+    {% assign language = site.language %}
+{% endif %}
+
+<html lang="{{ language }}">
 {% include head.html %}
 <body>
     {{ content }}
     {% include analytics.html %}
     <script defer type="text/javascript" src="{{ site.baseurl }}/static/js/index.js"></script>
     <script defer type="text/javascript" src="{{ site.baseurl }}/static/js/twitter.js"></script>
+    <script type="text/javascript" src="https://cdn.jsdelivr.net/docsearch.js/1/docsearch.min.js"></script>
```

Comment:
Please move the script tag into the included file, and add the `defer` property.